docs/UserGuide.adoc

```diff
@@ -287,7 +287,7 @@ A sample image of how the medicine inventory data in the exported CSV file is or
 image::SampleCSVFileImage.png[width="790"]
 
 [NOTE]
-From the image above, you can observe that different batches of the same medicine have their own row in the CSV file. This provides more detailed information on the quantity and expiry date for each batch of the same medicine as different batches of the same medicine can have different expiry dates and quantity. Furthermore, the CSV file also includes information such as which medicines are low in stock and which medicines are expiring soon for easier reference.
+From the image above, you can observe that different batches of the same medicine have their own row in the CSV file. This provides more detailed information on the quantity and expiry date for each batch of the same medicine as different batches of the same medicine can have different expiry dates and quantity. Furthermore, the CSV file also includes information such as which medicines are low in stock, which medicines are expiring soon or have expired for easier reference.
 
```

Comment:
Think it should be which medicines are low in stock, expiring soon or have expired

Reply:
Will edit that sentence in the next update.